python/kserve/kserve/model.py

```diff
@@ -139,19 +139,36 @@ def __init__(
         predictor_protocol: str = PredictorProtocol.REST_V1.value,
         predictor_use_ssl: bool = False,
         predictor_request_timeout_seconds: int = 600,
+        predictor_request_retries: int = 3,
```

Comment:
I think it is better to default to 0 for calling predictor as the client side might already have configured the retries

Reply:
done